spec/unit/recipes/source_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe 'prometheus::source' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.